test/testUtils.ts

```diff
@@ -0,0 +1,9 @@
+import t from 'tap'
+
+export async function throwsWithMessage(funcUnderTest: () => {}, expectedMessage: string) {
```

Comment:
What's the difference of this vs just using `t.throws`?

Reply:
`t.throws` wasn't checking the error message as well. (It might be that I was using it incorrectly.)

Reply:
I played around with a few different arguments to the `tap` call, but couldn't get it to match the error message thrown. I may be missing something obvious [in the docs](https://node-tap.org/docs/api/asserts/#tthrowsfn-expectederror-message-extra).